azure-iot-device/tests/iothub/test_sync_clients.py

```diff
@@ -200,37 +236,201 @@ def test_receive_method_request_default_mode(self, mocker, client, method_name):
         assert inbox_mock.get.call_count == 1
         assert inbox_mock.get.call_args == mocker.call(block=True, timeout=None)
 
-    def test_send_method_response_calls_transport(self, client, pipeline):
+    # TODO: add missing tests
+
+
+class SharedClientSendMethodResponseTests(object):
+    @pytest.mark.it("Begins a 'send_method_response' pipeline operation")
+    def test_send_method_response_calls_pipeline(self, client, pipeline):
         response = MethodResponse(request_id="1", status=200, payload={"key": "value"})
         client.send_method_response(response)
         assert pipeline.send_method_response.call_count == 1
         assert pipeline.send_method_response.call_args[0][0] is response
 
 
-@pytest.mark.describe("IoTHubModuleClient (Synchronous)")
-class TestIoTHubModuleClient(ClientSharedTests):
+################
+# DEVICE TESTS #
+################
+class IoTHubDeviceClientTestsConfig(object):
+    client_class = IoTHubDeviceClient
+
+    @pytest.fixture
+    def client(self, pipeline):
+        return IoTHubDeviceClient(pipeline)
+
+
+@pytest.mark.describe("IoTHubDeviceClient (Synchronous) - Instantiation")
+class TestIoTHubDeviceClientInstantiation(
```

Comment:
This class inherits from the SharedClientInstantiationTests that define most of the tests for inheritance, as well as the DeviceClientTestsConfig which provides a fixture. Then it is extended with a test specific to instantiation of a DeviceClient.